gui/.gitignore

```diff
@@ -0,0 +1,45 @@
+lib-cov
```

Comment:
What are these files? I really feel like all of these should be blanketed as ignored.